aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/AipConsoleServiceImpl.java

```diff
@@ -21,4 +27,25 @@ public ApiInfoDto getAipConsoleApiInfo() throws AipConsoleException {
             throw new AipConsoleException("Error while retrieving AIP Console capabilities", e);
         }
     }
+
+    @Override
+    public void updateModuleGenerationType(String applicationGuid, JobRequestBuilder builder, ModuleGenerationType moduleGenerationType, boolean firstVersion) {
+        if (moduleGenerationType != null) {
```

Comment:
i dont understand why u put this in aipconsoleservice, it is related to application, so should be inside applicationservice

Reply:
Yes sir. Moving to application service